plugins/circleci/src/pages/DetailedViewPage/lib/ActionOutput/ActionOutput.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState, FC, Suspense } from 'react';
+import {
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  Typography,
+  ExpansionPanelDetails,
+} from '@material-ui/core';
+import moment from 'moment';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { makeStyles } from '@material-ui/core/styles';
+import { BuildStepAction } from 'circleci-api';
+
+const LazyLog = React.lazy(() => import('react-lazylog/build/LazyLog'));
+moment.relativeTimeThreshold('ss', 0);
+const useStyles = makeStyles({
+  expansionPanelDetails: {
+    padding: 0,
+  },
+  button: {
+    order: -1,
+    marginRight: 0,
+    // FIXME: how not to hardcode this
+    marginLeft: '-20px',
+  },
+});
+
+export const ActionOutput: FC<{
+  url: string;
+  name: string;
+  className?: string;
+  action: BuildStepAction;
+}> = ({ url, name, className, action }) => {
+  const classes = useStyles();
+
+  const [messages, setMessages] = useState([]);
+  useEffect(() => {
+    fetch(url)
```

Comment:
probably use `useAsync` instead from `react-use`. Otherwise handle unmount with `let didCancel = false` + `return () { didCancel = true }` pattern

Reply:
Actually, this part we forgot to migrate into the redux setup

Reply:
@Nek ^